src/app/components/TradeTable/index.jsx

```diff
@@ -53,13 +54,28 @@ const TradeTable = ({ swaps, handleClick }) => (
   </Table>
 )
 
-const createStatusLabel = (status) => {
-  const icon = status == 'order_complete' ? (
-    <i style={{ fontSize: '18px' }} className='text-success fa fa-check-circle'></i>
-  ) : (
-    <i className='fa fa-spinner fa-pulse'/>
-  )
-  return <Expandable shrunk={icon} expanded={status.replace('_',' ')}></Expandable>
+//NOTE: if we end up not filtering out other statuses this approach wont work
+const statusInfo = {
+  support: {
+    icon: <i className='fa fa-exclamation-circle text-warning'></i>,
+    expanded: 'There may be an issue. Contact support@faa.st for more info.'
+  },
+  complete: {
+    icon: <i style={{ fontSize: '18px' }} className='text-success fa fa-check-circle'></i>,
+    expanded: 'Order Complete'
+  },
+  processing: {
+    icon: <i className='fa fa-spinner fa-pulse'/>,
+    expanded: 'Processing'
+  }
+}
+
+const createStatusLabel = (swap) => {
+  const { contactSupport, status: { detailsCode } } = swap
+  const { support, complete, processing } = statusInfo
+  const status = contactSupport ? support : detailsCode == 'order_complete' ? complete : processing
```

Comment:
You can simplify this by using `pending`, `failed`, and `complete` as your `statusInfo` keys and:
`const status = contactSupport ? statusInfo.support : statusInfo[status.code]`